keyboardprocessor/doc/introspection.schema

```diff
@@ -0,0 +1,61 @@
+{"$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "keyman/keyboardprocessor/doc/introspection.schema",
```

Comment:
Yes, this should be a URI to a web accessible copy of the schema.  This could be on github, gituhb.io or the official keyman site. What's currently here is best effort

Reply:
Fixed in fecd218b0c7eefe4539397a290d1ccce746a6f4a